### PR TITLE
Add Ton Macron administration

### PR DESCRIPTION
**Pull request by rpg600**

### CODE REVIEW

File: src/AppBundle/Admin/TonMacronFriendInvitationAdmin.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Show\ShowMapper;
+
+class TonMacronFriendInvitationAdmin extends AbstractAdmin
+{
```
Comment (tgalopin) [inline src/AppBundle/Admin/TonMacronFriendInvitationAdmin.php]: Can you put ordered by createdAt by default?

File: src/AppBundle/Admin/TonMacronChoiceAdmin.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+use AppBundle\Entity\TonMacronChoice;
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+
+class TonMacronChoiceAdmin extends AbstractAdmin
+{
+    protected function configureFormFields(FormMapper $formMapper)
```
Comment (tgalopin) [inline src/AppBundle/Admin/TonMacronChoiceAdmin.php]: Can you put ordered by createdAt by default?

Comment (rpg600) [inline src/AppBundle/Admin/TonMacronChoiceAdmin.php]: We don't have createdAt field, do i need to add one ?

Comment (tgalopin) [inline src/AppBundle/Admin/TonMacronChoiceAdmin.php]: No it's fine

File: src/AppBundle/Admin/TonMacronFriendInvitationAdmin.php (hunk 1 above)
Comment (hhamon) [inline src/AppBundle/Admin/TonMacronFriendInvitationAdmin.php]: I think it's better to order by step number first and sequence (content key) number then. Thus we can disable pagination for such a list. This way we could override the list template to have an output similar to the following one:

````
Étape 0 : choix hors étapes du tunnel Ton Macron
---------------------------------------------------------

Label | Contenu
------------ | -------------
Introduction | Contenu de l'introduction
Conclusion | Contenu de la conclusion

Étape 1 : situation personnelle de l'ami
---------------------------------------------

Label | Contenu
------------ | -------------
A | Contenu de A
B | Contenu de B
C | Contenu de C

Étape 2 : projet de l'ami
---------------------------

Label | Contenu
------------ | -------------
D | Contenu de D
E | Contenu de E
F | Contenu de F
```

Comment (tgalopin) [inline src/AppBundle/Admin/TonMacronFriendInvitationAdmin.php]: That's a very good idea, let's do this!

Comment (tgalopin) [inline src/AppBundle/Admin/TonMacronFriendInvitationAdmin.php]: Actually, it's not the right entity. You're right and we should do this on TonMacronChoiceAdmin.php, but this entity should be ordered by createdAt DESC.

**Pull request merged**